src/tests/functional/plugin/shared/include/behavior/compiled_model/device.hpp

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <gtest/gtest.h>
+
+#include <base/ov_behavior_test_utils.hpp>
+
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+#    include <iostream>
+#    define GTEST_COUT std::cerr << "[          ] [ INFO ] "
+#    include <codecvt>
+#    include <functional_test_utils/skip_tests_config.hpp>
+
+#endif
+
+namespace ov {
+namespace test {
+namespace behavior {
+
+class OVClassCompiledModelWithDeviceTest :
+        public OVClassNetworkTest,
+        public ::testing::WithParamInterface<std::string>,
+        public OVCompiledNetworkTestBase {
+protected:
+    std::string deviceName;
+
+public:
```

Comment:
`Setup` by default is protected and there is no need to make it public.
Can public be removed?